README.md

```diff
@@ -510,6 +510,7 @@ Hurray! You successfully made a contribution! 🎉
   - [Subhangini](https://github.com/Subhangini)
   - [Suhaas](https://github.com/suhaaskataria)
   - [Suhail Khan](https://github.com/suhail34)
+  - [Sumeet](https://github.com/LegendSumeet)
```

Comment:
Please add your name between Sumit Gaur and Sumeet Naik. 🙂
My bad 😅 I got confused with the order again. Sorry for the inconvenience.